packages/entity-list/src/components/ColumnPicker/ColumnPicker.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types'
```

Comment:
To one commit you added the changelog "Hide all columns that were set to hidden in the preferences" imo what counts is that you added the dialog. No need for to much detail when it comes to the changelog. (Just a detail you dont need to change that)